package.json

```diff
@@ -20,106 +20,105 @@
-        "util": "^0.12.3"
-    },
-    "scripts": {
-        "generate_api": "npm run patch && npm run download_api && npm run generate_enums && npm run generate_exports && npm run typegen",
-        "postinstall": "npm run generate_api",
-        "start": "npx webpack serve --config webpack.dev.js",
-        "build": "npx webpack --config webpack.prod.js",
-        "msbuild": "npm run build && node scripts/msbuild.js",
-        "build-dev": "npx webpack --config webpack.dev.js",
-        "build-github": "npx webpack --config webpack.github.js",
-        "lint": "eslint ./src --max-warnings=0",
-        "lint-fix": "eslint --fix ./src --max-warnings=0",
-        "patch": "node scripts/patcher/patch.js",
-        "download_api": "node scripts/api_download.js",
-        "generate_enums": "node scripts/generate_enums.js > src/ApiClient/generatedcode/_enums.ts",
-        "generate_exports": "node scripts/generate_exports.js > src/ApiClient/generatedcode/schemas.d.ts",
-        "typegen": "npx typegen src/ApiClient/generatedcode/swagger.json > src/ApiClient/generatedcode/_generated.d.ts",
-        "test": "npm run lint && npm run build"
+        "util": "^0.12.4"
     },
     "devDependencies": {
-        "@babel/core": "^7.13.15",
-        "@babel/plugin-proposal-class-properties": "^7.13.0",
-        "@babel/preset-react": "^7.13.13",
-        "@babel/preset-typescript": "^7.13.0",
-        "@pmmmwh/react-refresh-webpack-plugin": "^0.5.1",
-        "@types/loadable__component": "^5.13.1",
-        "@types/react": "^16.14.2",
+        "@babel/core": "^7.16.0",
+        "@babel/plugin-proposal-class-properties": "^7.16.0",
+        "@babel/plugin-transform-runtime": "^7.16.4",
+        "@babel/preset-env": "^7.16.4",
+        "@babel/preset-react": "^7.16.0",
+        "@babel/preset-typescript": "^7.16.0",
+        "@pmmmwh/react-refresh-webpack-plugin": "^0.5.2",
+        "@types/loadable__component": "^5.13.4",
+        "@types/node": "^16.11.9",
+        "@types/react": "^17.0.35",
         "@types/react-bootstrap": "^1.0.1",
-        "@types/react-dom": "^16.9.10",
-        "@types/react-router-dom": "^5.1.7",
-        "@types/react-transition-group": "^4.4.0",
+        "@types/react-dom": "^16.9.14",
+        "@types/react-router-dom": "^5.3.2",
+        "@types/react-transition-group": "^4.4.4",
         "@types/webpack": "^5.28.0",
         "@typescript-eslint/eslint-plugin": "^3.10.1",
         "@typescript-eslint/parser": "^3.10.1",
-        "autoprefixer": "^10.2.5",
+        "autoprefixer": "^10.4.0",
+        "babel-loader": "^8.2.3",
         "copy-webpack-plugin": "^8.1.1",
-        "css-loader": "^5.2.5",
-        "eslint": "^7.19.0",
-        "eslint-config-prettier": "^6.15.0",
+        "css-loader": "^5.2.7",
+        "eslint": "^7.32.0",
         "eslint-plugin-import": "^2.22.1",
-        "eslint-plugin-prettier": "^3.3.1",
-        "eslint-plugin-react": "^7.22.0",
+        "eslint-plugin-radar": "^0.2.1",
```

Comment:
Either enable it in eslintrc or uninstall it

Reply:
removed